common/models/user.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+var request = require('request');
+var config = require('../../config.json');
+
+var authDomain = config.authDomain;
+var clientId = config.clientId;
+var accessToken;
+var refreshToken;
+var expiresIn;
```

Comment:
Only variables that are meant to be in-memory and shared with various users need to be here.  Otherwise, these need to be included with their respective variables.  Otherwise, one user will see other users' tokens.